platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm_legacy_wires.sv.template

```diff
@@ -28,7 +28,7 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import ccip_if_pkg::*;
```

Comment:
Is this change related to the name => class change?

Reply:
And here.